test/models/repo_test.rb

```diff
@@ -4,34 +4,53 @@ class RepoTest < ActiveSupport::TestCase
   test '#sync_ranks' do
     Repo.new.send(:sync_ranks)
 
-    {jeremy: 1, david: 2, jose: 3, xavier: 3, vijay: 3}.each do |c, r|
+    {jeremy: 1, david: 2, jose: 2, xavier: 4, vijay: 4}.each do |c, r|
```

Comment:
This seems like a bug. Not introduced here cause it's observable on the [official site](https://contributors.rubyonrails.org).

I think it makes more sense that if multiple contributors share a rank the next contributor should be assigned the next rank i.e. this should be:

```suggestion
    {jeremy: 1, david: 2, jose: 2, xavier: 3, vijay: 3}.each do |c, r|
```